UI/visibility-checkbox.cpp

```diff
@@ -7,10 +7,17 @@
 
 VisibilityCheckBox::VisibilityCheckBox() : QCheckBox()
 {
-	checkedImage =
-		QPixmap::fromImage(QImage(":/res/images/visible_mask.png"));
-	uncheckedImage =
-		QPixmap::fromImage(QImage(":/res/images/invisible_mask.png"));
+	if (devicePixelRatio() >= 2) {
+		checkedImage =
+			QPixmap::fromImage(QImage(":/res/images/visible_mask@2x.png"));
```

Comment:
Style: These lines are longer than 80-characters. You will probably have to line break them differently, like so:
```
		checkedImage = QPixmap::fromImage(QImage(
				":/res/images/visible_mask@2x.png"));
```

Reply:
Will fix